cmd/juju/charmhub/download.go

```diff
@@ -37,12 +37,18 @@ by a specified name. Downloading for a specific base can be done via
 --base. --base can be specified using the OS name and the version of
 the OS, separated by @. For example, --base ubuntu@22.04.
 
+By default, the latest revision in the stable channel will be
+downloaded. To download the latest revision from another channel,
+use --channel. To download a specific revision, add the revision
+suffix to the charm name. For example, postgresql-345.
+
 Adding a hyphen as the second argument allows the download to be piped
 to stdout.
 `
 
 	downloadExamples = `
     juju download postgresql
+	juju download postgresql-345
```

Comment:
Looks like you're using a tab vs spaces here, but this should be:
```suggestion
    juju download postgresql --revision 345
```